lib/schema.js

```diff
@@ -13,6 +13,31 @@ const addSchema = Symbol('addSchema');
 const factorify = Symbol('factorify');
 const initActions = Symbol('initActions');
 const validate = Symbol('validate');
+const classifyDomains = Symbol('classifyDomains');
+
+const domainConstructors = { Enum, Flags };
+const classWrappers = {
+  Money: value => new Money(value),
+  Uint8Array: value => Uint8Array.from(value),
```

Comment:
Why is there no try-catch block for the `Uint8Array` and `Money`?
i.e. `Uint8Array.from(null)` will throw TypeError.